src/main/java/com/app/gamaacademy/cabrasdoagrest/bankline/service/UsuarioServiceImpl.java

```diff
@@ -86,4 +85,8 @@ public boolean validaLoginCpfUnicos(String login, String cpf) {
 		return repository.findByLoginOrCpfEquals(login, cpf) == null;
 	}
 
+	public void setRepository(UsuarioRepository usuarioRepository) {
+		this.repository = usuarioRepository;
+	}
+
```

Comment:
Remover.

Não precisa de set para o mockito funcionar.

https://youtu.be/4VmbETu-dcA?t=1335